pkg/rtc/participant.go

```diff
@@ -1486,7 +1486,11 @@ func (p *ParticipantImpl) subscriberRTCPWorker() {
 
 			pkts = append(pkts, sr)
 			sd = append(sd, chunks...)
-			batchSize = batchSize + 1 + len(chunks)
+			numItems := 0
+			for _, chunk := range chunks {
+				numItems += len(chunk.Items)
+			}
+			batchSize = batchSize + 1 + numItems
```

Comment:
Keeping the count matching the older scheme by adding up items inside the chunk so that the batching behaviour does not change.